data/exploits/capture/http/forms/grabforms.rb

```diff
@@ -22,7 +22,7 @@ def usage
 sitelist = ARGV.shift() || usage()
 output   = ARGV.shift() || usage()
 
-File.readlines(sitelist).each do |site|
+File.readlines(sitelist, mode: 'rb').each do |site|
```

Comment:
I don't believe we need to use `rb` for non-binary files, but let me know your thoughts 👀